### ISSUE TITLE
Update semantic HTML

### ISSUE THREAD

**Issue by timhitchins** [timhitchins/rose-city-resource#15]
In order to improve accessibility, the client code should be scoured to look for semantic HTML (and general semantics) improvements.  More on semantics can be seen here: 
https://developer.mozilla.org/en-US/docs/Glossary/Semantics
